README.md

```diff
@@ -75,7 +75,7 @@ These scripts can all be run from the root level of the repo:
 - `yarn deploy-demo`
   - Builds the doc site locally and deploys it to a branch-specific path on GitHub Pages. The terminal will display the URL where the demo was deployed to after it is done running.
 - `yarn release`
-  - Bumps package versions and tags a release commit. Read our [Release Process guide](/guides/RELEASE-PROCESS.md) for more info.
+  - Bumps package versions and tags a release commit. Read our [release guide on Confluence](https://confluence.cms.gov/display/HCDSG/Creating+a+code+release) for more info.
```

Comment:
I think the links generated by Confluence's share button are permalinks. If we used that, this link would work even if the name of the doc changed or it was moved